.circleci/config.yml

```diff
@@ -84,6 +84,54 @@ jobs:
             source env/bin/activate
             make entry-point-test
 
+  package_test:
+    working_directory: ~/nlp_primitives
+    parameters:
+      image_tag:
+        type: string
+        default: "python:3.8"
+    executor:
+      name: python
+      image_tag: << parameters.image_tag >>
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/nlp_primitives
+      - run :
+          name: "Install Requirements"
+          command: |
+            virtualenv env -q
+            source env/bin/activate
+            pip install -r requirements.txt
+            pip install -r complete-requirements.txt
+      - install_test_requirements
+      - run :
+          name: "Create Source Distribution"
+          command: |
+            virtualenv env -q
+            source env/bin/activate
+            python setup.py sdist
+      - run :
+          name: "Install Source Distribution in Virtual Environment"
+          command: |
+            virtualenv env -q
+            source env/bin/activate
+            NLP_PRIMITIVES_VERSION=$(python setup.py --version)
+            pip install dist/nlp_primitives-${NLP_PRIMITIVES_VERSION}.tar.gz
+      - run:
+          name: "Remove Repo Contents To Avoid Importing Source From Repo"
```

Comment:
Ah, @rwedge  just pointed out that instead of messing with sys path to prevent the repo code from being imported in this test, we can just cd into a different directory. Will do